docs/root/configuration/http_conn_man/headers.rst

```diff
@@ -118,15 +118,22 @@ should be replaced by backslash-double-quote (\").
 
 The following keys are supported:
 
-1. ``By`` The Subject Alternative Name (SAN) of the current proxy's certificate.
+1. ``By`` The Subject Alternative Name (URI type SAN) of the current proxy's certificate.
```

Comment:
I think the SAN abbreviation just adds confusion now, maybe leave it as `The Subject Alternative Name (URI type)`?

Also, since we're adding DNS type, I'm questioning choice of `By` being URI type (not change in this PR, but related to it).